### PR TITLE
Fix tooltip delay

### PR DESCRIPTION
**Pull request by manfredt27**
image

### CODE REVIEW

**Comment by BrayanZ** [comment]
each form should have the tooltip defined on the object, rather than just inject it on the view

**Comment by manfredt27** [comment]
Done

File: src/app/styles/form.scss [hunk 1]
```diff
@@ -56,3 +56,28 @@ label {
 .small-font {
   font-size: 125% !important;
 }
+
+._tooltip{
```
Comment (BrayanZ) [inline src/app/styles/form.scss]: why are you adding this underscore to the class?

Comment (manfredt27) [inline src/app/styles/form.scss]: Because there was already a class named `tooltip`, I think it's from Foundation

Comment (BrayanZ) [inline src/app/styles/form.scss]: hmm, ok, it's not a big deal.
just add an space before the `{`s

**Comment by manfredt27** [comment]
Done

File: src/app/forms/zag-format/zag-format-small.jade [hunk 2]
```diff
@@ -10,7 +10,8 @@ form(novalidate=true, name="uvpForm", ng-hide="ctrl.showUVP")
     .small-10.columns
       input#attribute(type="text", placeholder="best customer attribute", ng-model="ctrl.method.attribute", required=true)
   .row
-    span.has-tip(data-tooltip, aria-haspopup=true, title= "Our publisher network is the only one that helps digital journalists to create public interviews") Example
+    a._tooltip(title="{{ctrl.method.example}}")
+      span(title="") Example
```
Comment (BrayanZ) [inline src/app/forms/zag-format/zag-format-small.jade]: Does the spans need the title attribute?

Comment (manfredt27) [inline src/app/forms/zag-format/zag-format-small.jade]: It looks like they don't. I'll remove them

**Comment by manfredt27** [comment]
Done

File: src/app/forms/zag-format/zag-format-small.jade (hunk 2 above)
Comment (BrayanZ) [inline src/app/forms/zag-format/zag-format-small.jade]: Actually I think the whole span is unnecessary.

**Comment by BrayanZ** [comment]
Also add a little arrow on the tooltip pointing to "Example". You can do it using `._tooltip:before` and playing with transparent borders

**Comment by manfredt27** [comment]
Done

File: src/app/styles/form.scss [hunk 3]
```diff
@@ -56,3 +56,39 @@ label {
...
+  content: attr(title);
+  left: 20%;
+  padding: 5px 15px;
+  position: absolute;
+  z-index: 98;
+  width: 220px;
+  cursor: help;
+}
+
+._tooltip:hover:before{
```
Comment (BrayanZ) [inline src/app/styles/form.scss]: space! :D

Comment (manfredt27) [inline src/app/styles/form.scss]: I already added it XD

**Comment by BrayanZ** [comment]
ok, one last thing :)
move the css of the tooltip to his own file

**Comment by manfredt27** [comment]
Done

**Pull request merged**